### PR TITLE
Format neutral content headers handler

### PR DESCRIPTION
**Pull request by cstamas**
As original MavenHeadersHandler was not maven specific at all,
it adds usual HTTP response headers that is/should be common
for all.

http://bamboo.s/browse/NX3-OSSF483

### CODE REVIEW

**Comment by adreghiciu** [comment]
+1

File: components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentHeadersHandler.java [hunk 1]
```diff
@@ -10,18 +10,12 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.view;
```
Comment (jdillon) [inline components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentHeadersHandler.java]: There is a view.handlers package where standard handles are meant to live, why aren't you using that?

**Comment by kellyrob99** [comment]
+1

File: components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentHeadersHandler.java (hunk 1 above)
Comment (cstamas) [inline components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentHeadersHandler.java]: Uh oh, agreed, I just noticed there are a wealth of other handlers in this package too, will move 'em here too.

**Comment by cstamas** [comment]
Merged as 9eba276e7e52fbe1d5a7bf2765f7595133c63110, will move handler to their place directly on master

**Comment by cstamas** [comment]
0e1a0ea7c5a6f5de8034473a91e2c1c4023ac5ac commit moves all the handler to proper place.

**Pull request closed without merging**